### PR TITLE
Move Login business logic from gRPC API to Accountmanager

### PR DESCRIPTION
**Pull request by braginini**
## Describe your changes

The Management gRPC API has too much business logic happening while it has to be in the Account manager.
This also needs to make more requests to the store through the account manager.

### Checklist
- [ ] Is it a bug fix
- [ ] Is a typo/documentation fix
- [ ] Is a feature enhancement
- [x] It is a refactor
- [ ] Created tests that fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: management/server/account_test.go [hunk 1]
```diff
@@ -1018,7 +1014,7 @@ func TestAccountManager_UpdatePeerMeta(t *testing.T) {
 	}
 
 	assert.Equal(t, newMeta, p.Meta)
-}
+}*/
```
Comment (pascal-fischer) [inline management/server/account_test.go]: Commented test should be fixed or removed

**Review by pascal-fischer — APPROVED** [review]

**Review by pascal-fischer — DISMISSED** [review]

File: management/server/peer.go [hunk 2]
```diff
@@ -602,50 +617,173 @@ func (am *DefaultAccountManager) AddPeer(setupKey, userID string, peer *Peer) (*
...
+	if err != nil {
+		return nil, nil, status.Errorf(status.Unauthenticated, "peer is not registered")
+	}
+
+	err = am.checkPeerLoginExpiration(login.UserID, peer, account)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	peer = am.updatePeerMeta(peer, peer.Meta, account)
```
Comment (gigovich) [inline management/server/peer.go]: Is there a need `login.Meta` instead of `peer.Meta`?

Comment (braginini) [reply management/server/peer.go]: good catch

**Review by gigovich — APPROVED** [review]
LGTM

**Pull request merged**